velox/exec/NestedLoopJoinProbe.cpp

```diff
@@ -77,6 +78,50 @@ void NestedLoopJoinProbe::initialize() {
   joinNode_.reset();
 }
 
+void NestedLoopJoinProbe::initializeFilter(
```

Comment:
just moved this function up to make the code order more readable; no changes here.